CRITICAL - 2024-10-14 09:43:46 --> ErrorException: Trying to access array offset on value of type null
[Method: POST, Route: login]
in APPPATH\Controllers\Login.php on line 12.
 1 APPPATH\Controllers\Login.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'D:\\xampp\\htdocs\\API\\app\\Controllers\\Login.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 09:43:53 --> ErrorException: Trying to access array offset on value of type null
[Method: POST, Route: login]
in APPPATH\Controllers\Login.php on line 12.
 1 APPPATH\Controllers\Login.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'D:\\xampp\\htdocs\\API\\app\\Controllers\\Login.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 09:44:20 --> ErrorException: Trying to access array offset on value of type null
[Method: POST, Route: login]
in APPPATH\Controllers\Login.php on line 12.
 1 APPPATH\Controllers\Login.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'D:\\xampp\\htdocs\\API\\app\\Controllers\\Login.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 09:44:51 --> ErrorException: Trying to access array offset on value of type null
[Method: POST, Route: login]
in APPPATH\Controllers\Login.php on line 12.
 1 APPPATH\Controllers\Login.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'D:\\xampp\\htdocs\\API\\app\\Controllers\\Login.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 09:45:49 --> ErrorException: Trying to access array offset on value of type null
[Method: POST, Route: login]
in APPPATH\Controllers\Login.php on line 15.
 1 APPPATH\Controllers\Login.php(15): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'D:\\xampp\\htdocs\\API\\app\\Controllers\\Login.php', 15)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 09:47:48 --> Error: Cannot use object of type stdClass as array
[Method: POST, Route: login]
in APPPATH\Controllers\Login.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->index()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 09:52:53 --> CodeIgniter\HTTP\Exceptions\HTTPException: Failed to parse JSON string. Error: Syntax error
[Method: POST, Route: register]
in SYSTEMPATH\HTTP\IncomingRequest.php on line 537.
 1 SYSTEMPATH\HTTP\IncomingRequest.php(537): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidJSON('Syntax error')
 2 APPPATH\Controllers\Register.php(9): CodeIgniter\HTTP\IncomingRequest->getJSON()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Register->signup()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 10:01:20 --> ArgumentCountError: Too few arguments to function App\Controllers\Admin::adminupdate(), 0 passed in D:\xampp\htdocs\API\system\CodeIgniter.php on line 933 and exactly 1 expected
[Method: POST, Route: update/29]
in APPPATH\Controllers\Admin.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 10:01:29 --> ArgumentCountError: Too few arguments to function App\Controllers\Admin::adminupdate(), 0 passed in D:\xampp\htdocs\API\system\CodeIgniter.php on line 933 and exactly 1 expected
[Method: POST, Route: update/29]
in APPPATH\Controllers\Admin.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 10:02:50 --> ArgumentCountError: Too few arguments to function App\Controllers\Admin::adminupdate(), 0 passed in D:\xampp\htdocs\API\system\CodeIgniter.php on line 933 and exactly 1 expected
[Method: POST, Route: update/29]
in APPPATH\Controllers\Admin.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 10:03:02 --> ArgumentCountError: Too few arguments to function App\Controllers\Admin::adminupdate(), 0 passed in D:\xampp\htdocs\API\system\CodeIgniter.php on line 933 and exactly 1 expected
[Method: POST, Route: update/29]
in APPPATH\Controllers\Admin.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 10:03:38 --> Error: Call to undefined method App\Controllers\Admin::check()
[Method: POST, Route: update/29]
in APPPATH\Controllers\Admin.php on line 22.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('29')
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
ERROR - 2024-10-14 10:07:24 --> mysqli_sql_exception: Duplicate entry 'abvjcn' for key 'username' in D:\xampp\htdocs\API\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 D:\xampp\htdocs\API\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE api SET ...', 0)
#1 D:\xampp\htdocs\API\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET ...')
#2 D:\xampp\htdocs\API\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET ...')
#3 D:\xampp\htdocs\API\system\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET ...', Array)
#4 D:\xampp\htdocs\API\app\Models\UserModel.php(38): CodeIgniter\Model->__call('query', Array)
#5 D:\xampp\htdocs\API\app\Controllers\Admin.php(31): App\Models\UserModel->updateUser('29', Array)
#6 D:\xampp\htdocs\API\system\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('29')
#7 D:\xampp\htdocs\API\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#8 D:\xampp\htdocs\API\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\API\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\API\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\API\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 D:\xampp\htdocs\API\system\rewrite.php(44): require_once('D:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-10-14 10:07:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'abvjcn' for key 'username'
[Method: POST, Route: update/29]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 2 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 3 APPPATH\Controllers\Admin.php(31): App\Models\UserModel->updateUser('29', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('29')
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 10:07:24 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'abvjcn' for key 'username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET username = \'abvjcn\', name = \'bh@bc.vvom\', email = \'Abho\', password = \'1234\' WHERE id = \'29\'')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET username = \'abvjcn\', name = \'bh@bc.vvom\', email = \'Abho\', password = \'1234\' WHERE id = \'29\'')
 3 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 4 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 5 APPPATH\Controllers\Admin.php(31): App\Models\UserModel->updateUser('29', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('29')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 10:07:24 --> [Caused by] mysqli_sql_exception: Duplicate entry 'abvjcn' for key 'username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE api SET username = \'abvjcn\', name = \'bh@bc.vvom\', email = \'Abho\', password = \'1234\' WHERE id = \'29\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET username = \'abvjcn\', name = \'bh@bc.vvom\', email = \'Abho\', password = \'1234\' WHERE id = \'29\'')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET username = \'abvjcn\', name = \'bh@bc.vvom\', email = \'Abho\', password = \'1234\' WHERE id = \'29\'')
 4 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 5 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 6 APPPATH\Controllers\Admin.php(31): App\Models\UserModel->updateUser('29', [...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('29')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 10:37:52 --> ArgumentCountError: Too few arguments to function generate_jwt(), 1 passed in D:\xampp\htdocs\API\app\Controllers\Login.php on line 26 and exactly 2 expected
[Method: POST, Route: login]
in APPPATH\Helpers\jwt_helper.php on line 5.
 1 APPPATH\Controllers\Login.php(26): generate_jwt([...])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 11:44:23 --> Error: Call to undefined function App\Controllers\validate_jwt()
[Method: POST, Route: delete/1]
in APPPATH\Controllers\Admin.php on line 100.
 1 APPPATH\Controllers\Admin.php(70): App\Controllers\Admin->validateJWT()
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->admindelete('1')
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 11:49:02 --> ErrorException: Undefined variable $token
[Method: POST, Route: delete/1]
in APPPATH\Controllers\Admin.php on line 92.
 1 APPPATH\Controllers\Admin.php(92): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $token', 'D:\\xampp\\htdocs\\API\\app\\Controllers\\Admin.php', 92)
 2 APPPATH\Controllers\Admin.php(73): App\Controllers\Admin->validateJWT()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->admindelete('1')
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 11:49:19 --> ErrorException: Undefined variable $authHeader
[Method: POST, Route: delete/1]
in APPPATH\Controllers\Admin.php on line 92.
 1 APPPATH\Controllers\Admin.php(92): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $authHeader', 'D:\\xampp\\htdocs\\API\\app\\Controllers\\Admin.php', 92)
 2 APPPATH\Controllers\Admin.php(73): App\Controllers\Admin->validateJWT()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->admindelete('1')
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
ERROR - 2024-10-14 12:03:24 --> Authorization header missing.
CRITICAL - 2024-10-14 12:10:11 --> CodeIgniter\HTTP\Exceptions\HTTPException: Failed to parse JSON string. Error: Syntax error
[Method: POST, Route: update/41]
in SYSTEMPATH\HTTP\IncomingRequest.php on line 537.
 1 SYSTEMPATH\HTTP\IncomingRequest.php(537): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidJSON('Syntax error')
 2 APPPATH\Controllers\Admin.php(16): CodeIgniter\HTTP\IncomingRequest->getJSON()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('41')
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
ERROR - 2024-10-14 12:10:21 --> mysqli_sql_exception: Duplicate entry '' for key 'username' in D:\xampp\htdocs\API\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 D:\xampp\htdocs\API\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE api SET ...', 0)
#1 D:\xampp\htdocs\API\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET ...')
#2 D:\xampp\htdocs\API\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET ...')
#3 D:\xampp\htdocs\API\system\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET ...', Array)
#4 D:\xampp\htdocs\API\app\Models\UserModel.php(38): CodeIgniter\Model->__call('query', Array)
#5 D:\xampp\htdocs\API\app\Controllers\Admin.php(34): App\Models\UserModel->updateUser('41', Array)
#6 D:\xampp\htdocs\API\system\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('41')
#7 D:\xampp\htdocs\API\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#8 D:\xampp\htdocs\API\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\API\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\API\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\API\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 D:\xampp\htdocs\API\system\rewrite.php(44): require_once('D:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-10-14 12:10:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'username'
[Method: POST, Route: update/41]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 2 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 3 APPPATH\Controllers\Admin.php(34): App\Models\UserModel->updateUser('41', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('41')
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 12:10:21 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET username = NULL, name = \'sadmin@huib.in\', email = NULL, password = \'12345\' WHERE id = \'41\'')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET username = NULL, name = \'sadmin@huib.in\', email = NULL, password = \'12345\' WHERE id = \'41\'')
 3 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 4 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 5 APPPATH\Controllers\Admin.php(34): App\Models\UserModel->updateUser('41', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('41')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 12:10:21 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE api SET username = NULL, name = \'sadmin@huib.in\', email = NULL, password = \'12345\' WHERE id = \'41\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET username = NULL, name = \'sadmin@huib.in\', email = NULL, password = \'12345\' WHERE id = \'41\'')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET username = NULL, name = \'sadmin@huib.in\', email = NULL, password = \'12345\' WHERE id = \'41\'')
 4 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 5 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 6 APPPATH\Controllers\Admin.php(34): App\Models\UserModel->updateUser('41', [...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('41')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
ERROR - 2024-10-14 12:53:24 --> mysqli_sql_exception: Duplicate entry 'check' for key 'email' in D:\xampp\htdocs\API\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 D:\xampp\htdocs\API\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE api SET ...', 0)
#1 D:\xampp\htdocs\API\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET ...')
#2 D:\xampp\htdocs\API\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET ...')
#3 D:\xampp\htdocs\API\system\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET ...', Array)
#4 D:\xampp\htdocs\API\app\Models\UserModel.php(38): CodeIgniter\Model->__call('query', Array)
#5 D:\xampp\htdocs\API\app\Controllers\Admin.php(56): App\Models\UserModel->updateUser('36', Array)
#6 D:\xampp\htdocs\API\system\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
#7 D:\xampp\htdocs\API\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#8 D:\xampp\htdocs\API\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\API\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\API\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\API\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 D:\xampp\htdocs\API\system\rewrite.php(44): require_once('D:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-10-14 12:53:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'check' for key 'email'
[Method: POST, Route: update/36]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 2 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 3 APPPATH\Controllers\Admin.php(56): App\Models\UserModel->updateUser('36', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 12:53:24 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'check' for key 'email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET username = \'omfoov\', name = \'sadmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET username = \'omfoov\', name = \'sadmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'')
 3 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 4 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 5 APPPATH\Controllers\Admin.php(56): App\Models\UserModel->updateUser('36', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 12:53:24 --> [Caused by] mysqli_sql_exception: Duplicate entry 'check' for key 'email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE api SET username = \'omfoov\', name = \'sadmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET username = \'omfoov\', name = \'sadmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET username = \'omfoov\', name = \'sadmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'')
 4 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 5 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 6 APPPATH\Controllers\Admin.php(56): App\Models\UserModel->updateUser('36', [...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
ERROR - 2024-10-14 12:58:34 --> mysqli_sql_exception: Duplicate entry 'check' for key 'email' in D:\xampp\htdocs\API\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 D:\xampp\htdocs\API\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE api SET ...', 0)
#1 D:\xampp\htdocs\API\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET ...')
#2 D:\xampp\htdocs\API\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET ...')
#3 D:\xampp\htdocs\API\system\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET ...', Array)
#4 D:\xampp\htdocs\API\app\Models\UserModel.php(38): CodeIgniter\Model->__call('query', Array)
#5 D:\xampp\htdocs\API\app\Controllers\Admin.php(61): App\Models\UserModel->updateUser('36', Array)
#6 D:\xampp\htdocs\API\system\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
#7 D:\xampp\htdocs\API\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#8 D:\xampp\htdocs\API\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\API\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\API\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\API\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 D:\xampp\htdocs\API\system\rewrite.php(44): require_once('D:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-10-14 12:58:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'check' for key 'email'
[Method: POST, Route: update/36]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 2 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 3 APPPATH\Controllers\Admin.php(61): App\Models\UserModel->updateUser('36', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 12:58:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'check' for key 'email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET username = \'omfoov\', name = \'sadmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET username = \'omfoov\', name = \'sadmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'')
 3 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 4 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 5 APPPATH\Controllers\Admin.php(61): App\Models\UserModel->updateUser('36', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 12:58:34 --> [Caused by] mysqli_sql_exception: Duplicate entry 'check' for key 'email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE api SET username = \'omfoov\', name = \'sadmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET username = \'omfoov\', name = \'sadmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET username = \'omfoov\', name = \'sadmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'')
 4 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 5 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 6 APPPATH\Controllers\Admin.php(61): App\Models\UserModel->updateUser('36', [...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
ERROR - 2024-10-14 13:02:08 --> mysqli_sql_exception: Duplicate entry 'check' for key 'email' in D:\xampp\htdocs\API\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 D:\xampp\htdocs\API\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE api SET ...', 0)
#1 D:\xampp\htdocs\API\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET ...')
#2 D:\xampp\htdocs\API\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET ...')
#3 D:\xampp\htdocs\API\system\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET ...', Array)
#4 D:\xampp\htdocs\API\app\Models\UserModel.php(38): CodeIgniter\Model->__call('query', Array)
#5 D:\xampp\htdocs\API\app\Controllers\Admin.php(62): App\Models\UserModel->updateUser('36', Array)
#6 D:\xampp\htdocs\API\system\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
#7 D:\xampp\htdocs\API\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#8 D:\xampp\htdocs\API\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\API\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\API\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\API\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 D:\xampp\htdocs\API\system\rewrite.php(44): require_once('D:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-10-14 13:02:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'check' for key 'email'
[Method: POST, Route: update/36]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 2 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 3 APPPATH\Controllers\Admin.php(62): App\Models\UserModel->updateUser('36', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 13:02:08 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'check' for key 'email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET username = \'omfoov\', name = \'sadbfjvcmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET username = \'omfoov\', name = \'sadbfjvcmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'')
 3 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 4 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 5 APPPATH\Controllers\Admin.php(62): App\Models\UserModel->updateUser('36', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 13:02:08 --> [Caused by] mysqli_sql_exception: Duplicate entry 'check' for key 'email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE api SET username = \'omfoov\', name = \'sadbfjvcmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE api SET username = \'omfoov\', name = \'sadbfjvcmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE api SET username = \'omfoov\', name = \'sadbfjvcmin@shipglobal.in\', email = \'check\', password = \'12345\' WHERE id = \'36\'')
 4 SYSTEMPATH\Model.php(926): CodeIgniter\Database\BaseConnection->query('UPDATE api SET username = ?, name = ?, email = ?, password = ? WHERE id = ?', [...])
 5 APPPATH\Models\UserModel.php(38): CodeIgniter\Model->__call('query', [...])
 6 APPPATH\Controllers\Admin.php(62): App\Models\UserModel->updateUser('36', [...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 13:06:15 --> ErrorException: Undefined variable $decodedData
[Method: POST, Route: update/36]
in APPPATH\Controllers\Admin.php on line 30.
 1 APPPATH\Controllers\Admin.php(30): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $decodedData', 'D:\\xampp\\htdocs\\API\\app\\Controllers\\Admin.php', 30)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
CRITICAL - 2024-10-14 13:09:33 --> ErrorException: Attempt to read property "email" on bool
[Method: POST, Route: update/36]
in APPPATH\Controllers\Admin.php on line 32.
 1 APPPATH\Controllers\Admin.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "email" on bool', 'D:\\xampp\\htdocs\\API\\app\\Controllers\\Admin.php', 32)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->adminupdate('36')
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('D:\\xampp\\htdocs\\API\\public\\index.php')
